prime-angular-frontend/src/app/modules/enrolment/pages/health-authority/health-authority.component.ts

```diff
@@ -105,12 +105,12 @@ export class HealthAuthorityComponent extends BaseEnrolmentProfilePage implement
   }
 
   protected nextRouteAfterSubmit() {
-    const jobs = this.enrolmentFormStateService.jobsForm.get('jobs').value as Job[];
+    const oboSites = this.enrolmentFormStateService.jobsForm.get('oboSites').value as OboSite[];
 
     let nextRoutePath: string;
     if (!this.isProfileComplete) {
       nextRoutePath = EnrolmentRoutes.REGULATORY;
-    } else if (jobs.length) {
+    } else if (oboSites.length) {
```

Comment:
You likely need a null check on this since you're doing it the care-setting.component - `oboSites?.length`

![image](https://user-images.githubusercontent.com/599498/116452665-cdc43e00-a812-11eb-8fda-1745009fab33.png)

Reply:
Addressed